com.github.stsdc.monitor.json

```diff
@@ -0,0 +1,347 @@
+{
+  "app-id": "com.github.stsdc.monitor",
+  "runtime": "io.elementary.Platform",
+  "runtime-version": "7.3",
+  "sdk": "io.elementary.Sdk",
+  "command": "com.github.stsdc.monitor",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=session-bus",
+    "--socket=system-bus",
```

Comment:
`session-bus` and `system-bus` are not allowed. You need to list the talk name that are needed.